lib/capability-tool.js

```diff
@@ -0,0 +1,280 @@
+/**
+* Class for checking capability and resource support in a capability statement
+*
```

Comment:
I think here would should type out that the expectation is for developers to use the porcelain, higher-level methods and list them out here.